### PR TITLE
Update name to PhoenixSlimeExample

### PR DESCRIPTION
**Pull request by doomspork**
@lpil your feedback is welcome but **do not merge** this one.  We need to ship the PhoenixSlime package so I can update `mix.exs` and `mix.lock`.

### CODE REVIEW

**Comment by doomspork** [comment]
I'll go through and rename the extensions to `.slime`.

**Comment by doomspork** [comment]
@lpil any thoughts?

File: web/templates/layout/app.html.slime [hunk 1]
```diff
@@ -16,5 +16,5 @@ html lang="en"
             a href="http://www.phoenixframework.org/docs"  Get Started
         span.logo
       = render "_flashes.html", conn: @conn
-      = @inner
+      = render @view_module, @view_template, assigns
```
Comment (lpil) [inline web/templates/layout/app.html.slime:19]: Why has this changed?

Comment (doomspork) [inline web/templates/layout/app.html.slime:19]: I updated the dependency versions and there was a Phoenix change:

```
** (ArgumentError) @inner has been removed in favor of explicit rendering with @view_module and @view_template assigns. Update your `<%= @inner %>` code to use `render/3`:

<%= render @view_module, @view_template, assigns %>
```

Comment (doomspork) [inline web/templates/layout/app.html.slime:19]: Phoenix Changelog: https://github.com/phoenixframework/phoenix/blob/master/CHANGELOG.md#110-2015-12-16

Comment (lpil) [inline web/templates/layout/app.html.slime:19]: That's a pretty cool change.

**Comment by lpil** [comment]
Looks good! :)

**Pull request merged**